codalab/model/bundle_model.py

```diff
@@ -1056,6 +1068,10 @@ def bundle_checkin(self, bundle, worker_run, user_id, worker_id):
             if not row:
                 return False
 
+            # Get staged bundle from worker checkin and move it to staged state
+            if worker_run.state == State.STAGED:
```

Comment:
Why wasn't this needed here before?

Reply:
What do you mean? I am not sure how it was done before. I am checking the `worker_run` state here to transition bundle to `staged` state. Maybe I missed some context?